sdcm/cluster.py

```diff
@@ -1408,23 +1408,26 @@ def config_setup(self, seed_address=None, cluster_name=None, enable_exp=True, en
         if server_encrypt or client_encrypt:
             self.remoter.send_files(src='./data_dir/ssl_conf',
                                     dst='/tmp/')
-            self.remoter.run('sudo mv /tmp/ssl_conf/*.* /etc/scylla/')
+            self.remoter.run('mkdir -p ~/.cassandra/')
+            self.remoter.run('cp /tmp/ssl_conf/unittest/cqlshrc ~/.cassandra/')
```

Comment:
why the `unittest` directory is in the path??

Reply:
There are some key files that generated from scylla unittest, the keys are used in client side.
Maybe just rename it to `ssl_conf/client/..`